src/components/SuperDrawer/useSuperDrawer.test.tsx

```diff
@@ -149,6 +149,17 @@ describe("useSuperDrawer", () => {
     // Then we called the callback
     expect(onClose).toHaveBeenCalledTimes(1);
   });
+
+  it("should not throw when attempting to closeDrawer with an empty content stack", () => {
+    // Given a useSuperDrawer and BeamContext hook
+    const hook = renderHook(useSuperDrawer, { wrapper }).result.current;
+    // And a closed SuperDrawer
+    // When we call onClose
+    // Then we do not expect to have an error thrown
+    act(() => {
+      expect(hook.closeDrawer).not.toThrow("Cannot read property 'kind' of undefined");
```

Comment:
Just curious, but if this threw "throw new Error(bad)" wouldn't that still pass this assertion, b/c the threw something that was not "Cannot read property..."? 

Like maybe just `.not.toThrow()` would be safer?

Seems like I should know off the top of my head, but I'm not sure...

Reply:
That's a good call... the type error is the specific one I was having an issue with, but surfacing any other unexpected ones is safer for sure.